src/loader/bgw_launcher.c

```diff
@@ -84,6 +84,8 @@ typedef enum SchedulerState
 
 static volatile sig_atomic_t got_SIGHUP = false;
 
+int ts_guc_bgw_scheduler_restart_time = 30;
```

Comment:
I assume these are seconds, but it might be good to add a comment about the dimension of this variable.